components/02-molecules/modal/_modal.scss

```diff
@@ -0,0 +1,95 @@
+// Base modal styles. Additional classes accomodate for custom app implementations.
+.modal__toggle {
+  cursor: pointer;
+}
+
+.modal__pane {
+  background-color: clr(background-section);
+  color: clr(text);
+  left: 0;
+  width: 100vw;
+  opacity: 0;
+  position: fixed;
+  top: 50%;
+  transform: translate(0%, -50%);
+  transition: opacity 300ms;
+  padding: $space;
+  visibility: hidden;
+  z-index: 1000;
```

Comment:
I order styles alphabetically, but I haven't checked to see what the latest is on that -- I've seen reference to "position styles first." Do you have a preference?

Reply:
I'm not actually sure what I prefer but agree with your comment here that there should be some sort of ordering. I updated the properties to be alphabetical.